README.md

```diff
@@ -341,18 +333,6 @@ docker build -f docker/Dockerfile -t graphenex .
 docker run -it --rm --name graphenex -p 8080:8080 --privileged graphenex
 ```
 
-## Screenshots
-
-![Screenshot I](https://user-images.githubusercontent.com/24392180/60615641-47866e80-9dd8-11e9-92fe-dfa8874db8fd.jpg)
-
-![Screenshot II](https://user-images.githubusercontent.com/24392180/60601258-7e4d8c00-9dba-11e9-93e1-96411b76b462.png)
-
-![Screenshot III](https://user-images.githubusercontent.com/24392180/60625283-9be51a80-9de7-11e9-99ea-46875d8b3ed8.JPG)
-
-![Screenshot IV](https://user-images.githubusercontent.com/24392180/60625314-bae3ac80-9de7-11e9-98ef-e5fa69c3be38.JPG)
-
-![Screenshot V](https://user-images.githubusercontent.com/24392180/60625373-eb2b4b00-9de7-11e9-9e9a-ab1869af9b1a.JPG)
-
```

Comment:
Let's add these back :) There is some nostalgic value attached to these.

Reply:
Sure.